webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityHeadSpace.vue

```diff
@@ -52,6 +60,9 @@ export default {
       return `${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/spaces/default-image/avatar`;
     },
     url() {
+      if (this.testvisibility) {
+        return null ;
+      }
       if (!this.groupId) {
         return '#';
       }
```

Comment:
could we have something like this :
```suggestion
      if (this.testvisibility || !this.groupId) {
        return '#';
      }
```

Reply:
Yes !